Mimir.Worker.Tests/Handler/AvatarStateHandlerTests.cs

```diff
@@ -0,0 +1,21 @@
+namespace Mimir.Worker.Tests.Handler;
+
+using Libplanet.Crypto;
+using Mimir.Worker.Handler;
+using Nekoyume.Model.State;
+using Xunit;
+
+public class AvatarStateHandlerTests
+{
+    private readonly AvatarStateHandler _handler = new AvatarStateHandler();
+
+    [Fact]
+    public void ConvertToStateData()
+    {
+        var address = new Address("4b4eccd6c6b17fe8d4312a0d2fadb0c93ad5a7ba");
+        var rawState = TestHelpers.ReadTestData("avatarState.txt");
+        var stateData = _handler.ConvertToStateData(address, rawState);
+
+        Assert.IsType<AvatarState>(stateData.State);
```

Comment:
Is it okay not to check the inner variables?

Reply:
I just used example code, so I didn't think too deeply about it.
It seems enough that check `AvatarState` type.

Reply:
Ah, if all you want to do is set up a project, that's fine 👍🏻